x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/configure_fields.tsx

```diff
@@ -11,37 +11,62 @@ import { useValues } from 'kea';
 
 import { EuiForm, EuiSpacer, EuiTitle, EuiText, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 
-import { i18n } from '@kbn/i18n';
-
 import { InferenceConfiguration } from './inference_config';
 import { MLInferenceLogic } from './ml_inference_logic';
 import { MultiFieldMapping, SelectedFieldMappings } from './multi_field_selector';
 import { SingleFieldMapping } from './single_field_selector';
+import { FormattedMessage } from '@kbn/i18n-react';
 
 export const ConfigureFields: React.FC = () => {
-  const { isTextExpansionModelSelected } = useValues(MLInferenceLogic);
+  const {
+    isTextExpansionModelSelected,
+    addInferencePipelineModal: { configuration },
+  } = useValues(MLInferenceLogic);
+  const areInputsDisabled = configuration.existingPipeline !== false;
```

Comment:
I'd like to turn this boolean into a property that is passed to this component (in a future PR).